weasyl/define.py

```diff
@@ -606,7 +608,10 @@ def iso8601_date(target):
     :param target: The target Weasyl timestamp to convert.
     :return: An ISO 8601 string representing the date of `target`.
     """
-    date = datetime.datetime.utcfromtimestamp(target - _UNIXTIME_OFFSET)
+    if isinstance(target, datetime.datetime):
+        date = target
+    else:
+        date = datetime.datetime.utcfromtimestamp(target - _UNIXTIME_OFFSET)
```

Comment:
It looks like this is part of the removed birthday change?

```suggestion
    date = datetime.datetime.utcfromtimestamp(target - _UNIXTIME_OFFSET)
```

Reply:
Yup!